py/dml/dmlc.py

```diff
@@ -233,6 +233,25 @@ def main(argv):
         '--dep', dest = "makedep", action = 'store',
         help = 'generate makefile dependencies')
 
+    # <dt>--no-dep-phony</dt>
```

Comment:
`--no-dep-phony` or `--no-phony-dep`?

Reply:
I'd prefer --no-phony-dep or possibly --dep-no-phony (as it's a sub-option of --dep)

Reply:
@mandolaerik Opinions?

Reply:
The reasoning here is that we can inverse the defaults eventually: make `no` the default and let gmake pass `--dep-phony`. The `--no-dep-phony` name then comes from the convention that the inverse of `--X` is named `--no-X`.

`--no-phony-dep` would be consistent with this. I welcome suggestions for better names, though; "phony dep" is not overwhelmingly descriptive.

Reply:
I second @mandolaerik here. `--dep-phony` is related to `--dep` and would add phony targets. `--no-dep-phony` is a reversal.

Reply:
fair enough, issue resolved I guess